ante/tests/setup.go

```diff
@@ -100,16 +99,22 @@ func (s *AnteTestSuite) CreateTestAccounts(numAccs int) []TestAccount {
 		priv, _, addr := testdata.KeyTestPubAddr()
 		acc := s.app.AccountKeeper.NewAccountWithAddress(s.ctx, addr)
 		err := acc.SetAccountNumber(uint64(i))
-		s.Require().NoError(err)
+		if err != nil {
```

Comment:
Those panic exits will need to be covered in a Ginkgo container to be able to replace testify assertions.
You can try adding them in a `BeforeSuite` container or add Ginkgo assertions in the setup function so that it can be called from child nodes.

Reply:
ginkgo assertions you mean the goOmega matchers right ?